mp/src/game/server/momentum/mom_triggers.cpp

```diff
@@ -1562,6 +1563,11 @@ bool CNoGrenadesZone::IsInsideNoGrenadesZone(CBaseEntity *pOther)
 {
     if ( pOther )
     {
+        // feels kinda hacky but not sure how else to get user buttons
+        const auto *pPlayer = dynamic_cast<CMomentumPlayer*>(pOther->GetOwnerEntity());
+        if (!pPlayer)
+        { return false; }
```

Comment:
```suggestion
            return false;
```

No need for braces on early returns.